src/Elmish/Component.purs

```diff
@@ -55,9 +86,10 @@ infix 8 mapCmds as <$$>
 mapCmds :: forall m msg innerMsg. Functor m => (msg -> innerMsg) -> Array (m msg) -> Array (m innerMsg)
 mapCmds mapMsg cmds = map mapMsg <$> cmds
 
--- | Creates a `Transition` without any commands
+-- | Creates a `Transition` without any commands.
+-- | This function will be deprecated soon in favor of `pure`.
```

Comment:
You could start marking this in the CHANGELOG using `Deprecated `: https://keepachangelog.com/en/1.0.0/#types

Reply:
Good point, totally missed it